app/views/teams/dashboard.html.haml

```diff
@@ -9,4 +9,4 @@
   = render 'summary', presenter: @presenter.team_stats_presenter
 
 :javascript
-  var lineChartData = #{raw @presenter.to_json(graphs: [{ type: :line_chart, data: :team_data }])};
+  lineChartData = #{raw @presenter.to_json(graphs: [{ type: :line_chart, data: :team_data }])};
```

Comment:
How come we don't need to set the units and labels here?

Reply:
@barbarawuk in presenter initializer

https://github.com/sardjv/murfin_method/blob/master/app/controllers/teams_controller.rb#L33

we use those methods with default values

https://github.com/sardjv/murfin_method/blob/master/app/controllers/concerns/remember_params.rb#L12

Reply:
it takes from cookie or from params if not found default one